neco_test.go

```diff
@@ -21,7 +21,6 @@ func TestNecoStartupScriptBuilder(t *testing.T) {
 
 	s := builder.Build()
 	shouldContain := []string{
-		"mkfs -t ext4 -F /dev/disk/by-id/google-local-ssd-0",
```

Comment:
Is this line disappeared from the new implementation? I suspect there would be nvme0n1 instead.

Reply:
Yes.  But I cannot find a good reason to test that.

Reply:
Anyways, it works.  I am using a locally patched version :-)

Reply:
I see, thanks for clarification.